runtime/doc/builtin.txt

```diff
@@ -79,6 +104,12 @@ append({lnum}, {text})                                                *append()*
 			let failed = append(0, ["Chapter 1", "the beginning"])
 <
 
+                Parameters:
+                  • lnum (`integer`)
+                  • text (`any`)
```

Comment:
A lot of these types still need refining. Maybe it's worth taking a pass at that during or before this?

Reply:
Does that need to block this? Those types are in the meta files already...

Reply:
No. Just a light suggestion.